docs/UserGuide.md

```diff
@@ -59,12 +59,16 @@ UniVerse is more than just a **desktop app for managing contacts**—it is a pla
 - Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
+- Note that name cannot include prefixes that are already part of our commands.
+
 - Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
 - Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
+- Work experience parameter `[w/WORK_EXPERIENCE]…​` can only be used one time. <br>
```

Comment:
remove the '...' after [w/WORK_EXPERIENCE] as it means that there can be more than one work experience for a contact

Reply:
Fixed